README.md

```diff
@@ -66,16 +69,18 @@ If you are planning on indexing GitHub issues in addition to the codebase, you w
 
 <details open>
 <summary><strong>:computer: Running locally</strong></summary>
-<p>To index the codebase:</p>
+<p>To index the codebase, run this command. This should take a few minutes, depending on the repo size.</p>
 
-    index github-repo-name \  # e.g. Storia-AI/repo2vec
+    # any repo in the format ORG_NAME/REPO_NAME
+    index Storia-AI/repo2vec \
         --embedder-type=marqo \
         --vector-store-type=marqo \
         --index-name=your-index-name
 
-<p> To chat with your codebase:</p>
+<p> To chat with your codebase, run this command. This should take around a minute, depending on the repo size.</p>
 
-    chat github-repo-name \
+    # any repo in the format ORG_NAME/REPO_NAME
+    chat Storia-AI/repo2vec \
```

Comment:
I'm wondering now if people will get confused because we're using exactly *this* repo, thinking that it's a strict requirement for the argument to be set to that.

Maybe we should do:
```
export GITHUB_REPO=...
export INDEX_NAME=...
chat $GITHUB_REPO \
   --vector-store-type=marqo \
   --index-name=$INDEX_NAME \
   ...
```
WDYT?

Reply:
Yeah that's fair! I'll defer to you on this, my personal thoughts are that people would be okay with changing this once they've tried it out

Reply:
Well let's try it this way, see if anybody complains :)